src/LiveComponent/tests/Unit/Twig/TemplateCacheWarmerTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Tests\Unit\Twig;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Cache\Adapter\NullAdapter;
+use Symfony\Component\Cache\Adapter\PhpArrayAdapter;
+use Symfony\UX\LiveComponent\Twig\TemplateCacheWarmer;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ */
+final class TemplateCacheWarmerTest extends TestCase
+{
+    public function testWarmUpCreatesCacheFile(): void
+    {
+        $templateCacheWarmer = $this->createTemplateCacheWarmer();
+        $cacheDir = sys_get_temp_dir();
```

Comment:
This could be in `setUp` :)

Reply:
Hmm.. i'm not really fan of _not_ instanciating the object i test in the test method :| 

You feel it d be really better ?

Reply:
No, I mean setting cacheDir ;)

Reply:
In the end i rewrote it... and i kinda like it, thanks :))

Reply:
Looks good 👍

Reply:
😅